CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Coding guidelines
```

Comment:
I'm not objecting to your suggestions based on your background don't take it personally. I'm just saying people who contribute to a programming language (very, very, very intimidating) are gonna be different who contribute to a MUD client (very intimidating). It's certainly true that Rust would like contributors of all sorts but I think in practice what they attract will be higher-grade caliber programmers compared to what we'll attract. Your background in fact is very much appreciated here!

I've added Feature Requests - let me know what you think of it now.